x-pack/plugins/apm/public/components/shared/LocalUIFilters/index.tsx

```diff
@@ -37,13 +40,19 @@ const LocalUIFilters = ({
   filterNames,
   children,
   showCount = true,
+  onFiltersLoad = noop,
 }: Props) => {
-  const { filters, setFilterValue, clearValues } = useLocalUIFilters({
+  const { filters, setFilterValue, clearValues, status } = useLocalUIFilters({
     filterNames,
     projection,
     params,
   });
 
+  useEffect(() => {
+    onFiltersLoad(filters);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [status]);
```

Comment:
I'm not sure this is the best way to access the filters. Did you try:
```js
const { uiFilters } = useUrlParams();
```

?

Reply:
... Okay, seems like you are not interested in the ui filter values but in the options. My mind is a little fuzzy on this: where are the options set?

Reply:
actually it's using the same values for visitor breakdown, maybe it should be a separate API i guess. 

![image](https://user-images.githubusercontent.com/3505601/87360124-b8bf3c80-c569-11ea-9b29-03c84d476172.png)